x-pack/plugins/security_solution/public/cloud_security_posture/components/csp_details/vulnerabilities_findings_details_table.tsx

```diff
@@ -221,13 +228,17 @@ export const VulnerabilitiesFindingsDetailsTable = memo(({ value }: { value: str
         </SecuritySolutionLinkAnchor>
         <EuiSpacer size="xl" />
         <DistributionBar
-          stats={getVulnerabilityStats({
-            critical: CRITICAL,
-            high: HIGH,
-            medium: MEDIUM,
-            low: LOW,
-            none: NONE,
-          })}
+          stats={getVulnerabilityStats(
```

Comment:
Same opinion here, we should be passing the functions as parameters and not part of the object.

Reply:
wdyt?

Reply:
Apologies,  I did not realize this was an array of objects.